Fundamentals/Backoffice/LogViewer/index.md

```diff
@@ -42,125 +42,97 @@ With the flexibility of Umbraco, we give you the power to implement your own `IL
-        {
-            // This method will not be called - as we have indicated that this 'CanHandleLargeLogs'
-            throw new NotImplementedException();
-        }
-        protected override IReadOnlyList<LogEvent> GetLogs(LogTimePeriod logTimePeriod, ILogFilter filter, int skip, int take)
-        {
-            var cloudStorage = CloudStorageAccount.Parse("DefaultEndpointsProtocol=https;AccountName=ACCOUNT_NAME;AccountKey=KEY;EndpointSuffix=core.windows.net");
-            var tableClient = cloudStorage.CreateCloudTableClient();
-            CloudTable table = tableClient.GetTableReference("LogEventEntity");
-
-            var logs = new List<LogEvent>();
-            var count = 0;
-
-            // Create the table query
-            // TODO: Use a range query to filter by start & end date on the Timestamp
-            TableContinuationToken token = null;
-            var results = new List<LogEventEntity>();
-            do
-            {
-                var queryResult = table.ExecuteQuerySegmentedAsync(new TableQuery<LogEventEntity>(), token).GetAwaiter().GetResult();
-                results.AddRange(queryResult.Results);
-                token = queryResult.ContinuationToken;
-            } while (token != null);
-
-            // Loop through the results, displaying information about the entity.
-            foreach (var entity in results.Skip(skip))
-            {
-                // Reads the compact JSON format stored in the 'Data' column back to a LogEvent
-                // Same as the JSON txt files does
-                var logItem = LogEventReader.ReadFromString(entity.Data);
-
-                if (count > skip + take)
-                {
-                    break;
-                }
-
-                if (count < skip)
-                {
-                    count++;
-                    continue;
-                }
-
-                if (filter.TakeLogEvent(logItem))
-                {
-                    logs.Add(logItem);
-                }
-
-                count++;
-            }
-
-            return logs;
-        }
-
-        public override IReadOnlyList<SavedLogSearch> GetSavedSearches()
-        {
-            //TODO: Fetch from Azure Table
-            return base.GetSavedSearches();
-        }
-
-        public override IReadOnlyList<SavedLogSearch> AddSavedSearch(string name, string query)
-        {
-            //TODO: Add to Azure Table
-            return base.AddSavedSearch(name, query);
-        }
-
-        public override IReadOnlyList<SavedLogSearch> DeleteSavedSearch(string name, string query)
-        {
-            //TODO: Remove from Azure Table
-            return base.DeleteSavedSearch(name, query);
-        }
+        //TODO: Add to Azure Table
+        return base.AddSavedSearch(name, query);
     }
+
+    public override IReadOnlyList<SavedLogSearch>? DeleteSavedSearch(string? name, string? query)
+    {
+        //TODO: Remove from Azure Table
```

Comment:
//Optional: If you want to store saved searches in the Azure Table Storage, implement here a method to remove from the Azure Table.